### PR TITLE
Feature/5127 create basic preview page

### PR DESCRIPTION
**Pull request by alibombaye**
Story [AB#5127](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/5127)
Task [AB#7869](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7869) [AB#7872](https://buyingcatalog.visualstudio.com/c5f97979-5b03-4d10-ba8d-871d0526b408/_workitems/edit/7872)

Screenshot 2020-06-19 at 13 40 28

### CODE REVIEW

File: app/pages/preview/contextCreator.test.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
+import manifest from './manifest.json';
+import { getContext } from './contextCreator';
+import { baseUrl } from '../../config';
+
+describe('order summary preview contextCreator', () => {
+  describe('getContext', () => {
+    it('should return the backLinkText', () => {
+      const context = getContext({ orderId: 'order-1', orderData: { description: 'Some order description' } });
```
Comment (samMcCarthyBJSS) [inline app/pages/preview/contextCreator.test.js:8]: could this const be set at the top they're all the same

File: app/pages/preview/controller.js [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import { getEndpoint } from '../../endpoints';
+import { logger } from '../../logger';
+import { getContext } from './contextCreator';
+
+export const getOrder = async ({ orderId, accessToken }) => {
+  const getOrderEndpoint = getEndpoint({ api: 'ordapi', endpointLocator: 'getOrder', options: { orderId } });
+  const orderData = await getData({
+    endpoint: getOrderEndpoint, accessToken, logger,
+  });
+
```
Comment (samMcCarthyBJSS) [inline app/pages/preview/controller.js:11]: Should this have a logger in saying it's come back from ordapi

File: app/pages/preview/contextCreator.test.js (hunk 1 above)
Comment (alibombaye) [reply app/pages/preview/contextCreator.test.js:8]: 👍 good shout. I'll make this change in the next PR

File: app/pages/preview/controller.js (hunk 2 above)
Comment (alibombaye) [reply app/pages/preview/controller.js:11]: yes it should. I'll add a log message

**Pull request merged**